sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+
+## Deposit
+
+A deposit operation is initiated as a Bitcoin transaction with a special payload
+in an `OP_RETURN` output (Figure 1).  Because this is a Stacks-on-Bitcoin
+transaction, a miner in a subsequent tenure is required by the consensus rules
+to process it as part of producing the first block in its tenure.  The Stacks
+node records the existence of this deposit operation in the data space of the
+`.sbtc` smart contract, where it is marked as a _pending_ operation that
+Stackers must act upon.  Namely, the relevant parts of the transaction are
+extracted and translated in to Clarity values and made available via internal data maps.
+
+Stackers, via their signer daemons, already subscribe to the Stacks blockchain
+for events.  The act of writing new sBTC deposit state to the `.sbtc` contract
+generates such an event -- an event that alerts the signer daemons to begin a
+signing round to act on the deposit.
+
+In the happy path, at least 70% of Stackers agree that the deposit transaction is
+valid, and they generate a contract-call to the `.sbtc` contract to
+_authorize_ the deposit by minting the equivalent number of sBTC tokens to
+an account address of the user's choice.  Crucially, the transaction is signed 
+collectively by Stackers via a WSTS signing round -- it is authenticated via a
+Schnorr proof and the Stacker's aggregate public key.  This way, _only_ the
+Stackers can authorize the deposit, and only via a 70%+ majority vote.
+
+Once stackers generate the authorization transaction, they make it available to
+miners via their StackerDB instance, and require the current miner to include it
+in the next block they produce.  If the miner does not do so, then the Stackers
+will not sign the block; this all but compels the miner to include the
+transaction.
+
+The user must wait until their deposit transaction confirms on Bitcoin before the
+sBTC tokens materialize.  This is required because Stacks miners only process
+pending Stacks-on-Bitcoin transactions for Bitcoin blocks up to but _excluding_
+the Bitcoin block in which their `block-commit` landed.
+
+The act of fulfilling a deposit with an affirmative vote is effectively a promise
+made by Stackers to users to ensure that the sBTC minted wll remain fully backed
+by the BTC the user submitted.  This BTC may only be spent to fulfill withdraws,
+or to process a wallet hand-off at the end of a reward cycle.  Either way, the
+user's BTC will be spent by Stackers before they can reclaim it via the spending
+fulfillment script.
+
+![Figure 1](./001-sip-025-figure-1.png)
+
+_Figure 1: Protocol diagram of an sBTC deposit.  The user submits a Bitcoin
+transaction, and eventually, the equivalent number of sBTC tokens materialize in
+the account of the user's choosing_.
+
+### Deposit Bitcoin Transaction
+
+The deposit Bitcoin transaction contains the following data:
+
+* An `OP_RETURN` data output, which contains the account address to which sBTC
```

Comment:
Where should the specification for this go? Currently, the Alpha Romeo DR 0.1 has some specs: https://github.com/stacks-network/sbtc/blob/f4130c63ec5680921617ac4b910d72ffb4072eb5/sbtc-core/src/operations/op_return/deposit.rs#L4